app/controllers/staff/reports_controller.rb

```diff
@@ -11,6 +11,7 @@ def index
     current_user.service_owner? ? active_reports_with_organisations : active_reports
     current_user.service_owner? ? submitted_reports_with_organisations : submitted_reports
     current_user.service_owner? ? in_review_reports_with_organisations : in_review_reports
+    current_user.service_owner? ? awaiting_changes_reports_with_organisations : awaiting_changes_reports
```

Comment:
I'm finding this hard to understand at this point - would this be more readable:

```ruby
def index
  if current_user.service_owner?
    reports_for_service_owner
  else
    reports_for_delivery_partner
  end
end

private

def reports_for_service_owner
  active_report_with_organisations
  ...
end

..etc...
```